src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -0,0 +1,210 @@
+$stacking-order: (
+  output: 10,
+  input: 20,
+);
+
+$range-track-height: rem(4px);
+$range-thumb-size: rem(24px);
```

Comment:
for accessibility it's recommended that the area be 48px., have we experimented with that?

Reply:
Yes, you're right. The size is based on the native range slider. Turns out there is a browser-compatible way to style the native thumb, so maybe we should increase the size on both.

Reply:
[WCAG 2.1 has added target size guidance at the AAA level](https://www.w3.org/TR/WCAG21/#target-size) (44px), which is technically out of scope for what we're targeting, from a """compliance""" perspective, since we (and most orgs) are focusing on A and AA level. That said, it might be helpful to release the slider as is and get some usability feedback! We could always adjust the size and other factors later.